style-guide/styles/wds-components/_toggles.scss

```diff
@@ -0,0 +1,76 @@
+@import '../wds-variables/colors';
+
+.wds-switch {
```

Comment:
Is it a switch or a toggle? Is switch a subtype of a toggle? In the description, we use `toggle`, in the code we use `switch`. Can we use one?